### PR TITLE
read me edits

### PR DESCRIPTION
**Pull request by hassa963**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Course Simulator
 The program is a course simulator where users can create and take different courses and get evaluated for them. We chose to create this program as it felt relevant as we are all students and the subject matter felt familiar! Moreover, the program can be used as a way for students to create quizzes that they can test themselves for upcoming assessments with that allows them to monitor how they progress.
+The Tests cover 75% of all components but this includes portions that we would not be testing below is a picture showing the coverage of our tests:
+
+
+<img width="326" alt="image" src="https://github.com/CSC207-2023Y-UofT/course-project-held-together-by-duct-tape/assets/133050886/9ed685a5-66f9-4491-9c8a-33b3e41e4bbe">
+
```
Comment (jpmedinagl) [inline README.md:7]: That's good ! Thank you for adding test coverage.

File: README.md [hunk 2]
```diff
@@ -39,7 +45,7 @@ Tip, you can use terminal to operate MySQL, or you can choose to download MySQL
 
 - create table sessioncourse (CourseID Varchar(25), Question Varchar(250), Answer Varchar(250), Points int);
 
-- create table sessionstudent (StudentID Varchar(25), CourseID Varchar(25), CourseGrade float);
+- create table sessionstudent (StudentID Varchar(25), Password Varchar(25), CourseID Varchar(25), CourseGrade float);
```
Comment (jpmedinagl) [inline README.md:48]: Yes thank you for making this change.

**Review by jpmedinagl — APPROVED** [review]
Everything looks good. Minor changes to the README so I'll approve it.

**Review by FXXFERMI — APPROVED** [review]
Everything looks great! Thanks for changing them!

**Pull request merged**